eclipse-scout-core/test/action/ActionSpec.ts

```diff
@@ -140,6 +140,35 @@ describe('Action', () => {
 
   });
 
+  describe('tooltipText', () => {
+    it('is shown on mouseover after a short delay ', async () => {
+      let action = scout.create(Action, {
+        parent: session.desktop,
+        tooltipText: 'tooltip'
+      });
+      action.render();
+      JQueryTesting.triggerMouseEnter(action.$container);
+      expect(action.findChild(Tooltip)).toBeFalsy();
+
+      await sleep(tooltips.DEFAULT_TOOLTIP_DELAY);
```

Comment:
If I understand it correctly, this sleep() method sleeps in real-time. Therefore these two test cases delay the execution of all jasmine tests by over a second.

Could we use the jasmine clock here to speed this up? Otherwise consider shortening the delay for this "describe", e.g. by changing the value of the DEFAULT_TOOLTIP_DELAY constant or overwriting Action#_configureTooltip.